Src/BootCamp.Chapter/Shop.cs

```diff
@@ -1,28 +1,15 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Linq;
+
+namespace BootCamp.Chapter
 {
-    public class Shop
+    public class Shop : Inventory
```

Comment:
Shop is not an inventory. That's illogical. Don't inherit just because it's convenient for now.

Reply:
Ok. I now added new Inventory in the shop instead. However your ShopTest requires that shop has Items (_shop.Items.Should etc.).
The Inventory I added to shop already has Items as a prop but how can I make a reference to this in the Shop directly?
I solved this by adding Items also in the shop class and in the constructor put Items = inventory.Items. Ok?
Of course I had to add this line also everytime I add/remove something in inventory.Items

Reply:
exactly. Shop **is not** inventory. Sho **has** inventory!

Reply:
Thanks for review!